### PR TITLE
Luuk/orbiit surt font

### PR DESCRIPTION
**Pull request by luckylwk**

### CODE REVIEW

File: src/components/DesignProvider/DesignProvider.js [hunk 1]
```diff
@@ -17,16 +18,50 @@ const GlobalStyle = createGlobalStyle`
...
     color: ${({ theme }) => theme.colors.neutral.darker};
+
     text-align: left;
    }
 
   * {
     color: inherit;
   }
+
+  @font-face {
```
Comment (jonashaefele) [inline src/components/DesignProvider/DesignProvider.js:34]: I did this with Poppins originally, but Next.js gets angry if you use @import for fonts. Some vulnerability or something. Hence be building that `<Fonts />` component. A bit more annoying becaue you have to add both `DesignProvider` and `Fonts`, but the error went away.

Comment (luckylwk) [reply src/components/DesignProvider/DesignProvider.js:34]: Oh interesting, I haven't seen it throw that yet, but will try it by updating `Fonts`

**Comment by jonashaefele** [comment]
I think it's a warning still, not an error, it still builds. But it showed
up big and red in my console...
Jonas

*~*
Jonas Haefele
Founder of slow.works
Coaching and consulting in mindful and sustainable tech.
In order to focus on my work, I check emails twice a day and aim to respond
the next day.
In emergencies, please call +44 742 906 6868
*~*

**Comment by jonashaefele** [comment]
I can do a separate PR to harmonise font sizes again and check with the designs. Poppins and Surt render in different sizes.

File: package.json [hunk 2]
```diff
@@ -42,7 +42,13 @@
...
     "styled-components": "^5.2.0",
     "styled-system": "^5.0.6",
-    "unified": "^8.3.2"
+    "unified": "^8.3.2",
+    "node-forge": "^0.10.0",
+    "node-fetch": "^2.6.1",
+    "yargs-parser": "^13.1.2",
+    "http-proxy": "^1.18.1",
+    "dot-prop": "^4.2.1",
+    "serialize-javascript": "^3.1.0"
```
Comment (jonashaefele) [inline package.json:51]: what do all of these do?

Comment (luckylwk) [reply package.json:51]: Fix vulnerabilities

Comment (luckylwk) [reply package.json:51]: I keep getting github dependabot notifications

**Pull request merged**